lib/public/Files/SimpleFS/ISimpleFolder.php

```diff
@@ -22,6 +22,14 @@ interface ISimpleFolder {
 	 */
 	public function getDirectoryListing(): array;
 
+	/**
+	 * Get all the files and folders in a folder
+	 *
+	 * @return (ISimpleFile[] || ISimpleFolder[]) 
+	 * @since 30.0.5
```

Comment:
master branch is targeting 31, however we are in API freeze so should be 32 already.

```suggestion
	 * @since 32.0.0
```